File: app/src/main/java/com/example/msi/websocketdemo/websocket/WebSocketListener.java
package com.example.msi.websocketdemo.websocket;

import com.example.msi.websocketdemo.websocket.response.WebSocketMsg;

/**
 * Description:
 *
 * @author: guoyongping
 * @date: 2019-05-31 15:44
 */
public interface WebSocketListener {

    /**
     * 当接收到系统消息
     */
    void onMessageResponseClient(WebSocketMsg msg, int index);
}

File: app/src/main/java/com/example/msi/websocketdemo/websocket/WebSocketListener.kt
package com.example.msi.websocketdemo.websocket

import com.example.msi.websocketdemo.websocket.response.WebSocketMsg

/**
 * Description:
 *
 * @author: guoyongping
 * @date: 2019-05-31 15:44
 */
interface WebSocketListener {

    /**
     * 当接收到系统消息
     */
    fun onMessageResponseClient(msg: WebSocketMsg, index: Int)
}
